### PR TITLE
lint: markdown using prettier

### PR DESCRIPTION
**Pull request by maddhruv**
Lints all markdown files using prettier

Related - https://github.com/atlassian/changesets/issues/31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "lint:prettier": "prettier --list-different \"**/*.{js,ts,tsx}\"",
     "types:check": "tsc",
     "prettier": "prettier --write \"**/*.{js,ts,tsx}\"",
+    "prettier:md": "prettier --write \"**/*.md\"",
```
Comment (Andarist) [inline package.json:16]: Personally, I would prefer just adding this extension to the list in the `prettier` task. Similar should be done for the `lint:prettier` task as well.

Comment (maddhruv) [reply package.json]: Done 👍

**Review by Andarist — APPROVED** [review]

**Comment by maddhruv** [comment]
Thanks @Andarist 🤗

**Comment by Andarist** [comment]
Thank you!

**Pull request merged**